### PR TITLE
Dynamic gradient button

### PR DESCRIPTION
**Pull request by masterrom**
Gradient Button now contains the disabled option with custom styling

### CODE REVIEW

File: src/screens/setup/create/CreateWalletName.js [hunk 1]
```diff
@@ -14,6 +14,18 @@ const ethers = require('ethers');
  */
 class CreateWalletName extends Component {
 
+
+    constructor(props) {
+        super(props);
+        this.state = {
+
```
Comment (priom) [inline src/screens/setup/create/CreateWalletName.js]: reformat the code block to remove unnecessary whitespaces

File: src/components/LinearGradient/LinearButton.js [hunk 2]
```diff
@@ -38,6 +40,12 @@ class LinearButton extends Component {
     }
 }
 
+const gradientColors = {
+    Enabled: ['#04b79f', '#5cfab1'],
+    Disabled: ['#a2a3a5', '#e5e5e5']
+}
+
```
Comment (GregTheGreek) [inline src/components/LinearGradient/LinearButton.js]: white space

File: src/screens/setup/create/CreateWalletName.js [hunk 3]
```diff
@@ -14,6 +14,18 @@ const ethers = require('ethers');
  */
 class CreateWalletName extends Component {
 
+
+    constructor(props) {
```
Comment (GregTheGreek) [inline src/screens/setup/create/CreateWalletName.js]: white space

File: src/screens/setup/create/CreateWalletName.js [hunk 4]
```diff
@@ -14,6 +14,18 @@ const ethers = require('ethers');
...
+        super(props);
+        this.state = {
+            
+            buttonDisabled: true
+        
+        }
+    }
+
+
```
Comment (GregTheGreek) [inline src/screens/setup/create/CreateWalletName.js]: white space

**Pull request merged**